.github/workflows/kb_sdk-actions.yml

```diff
@@ -0,0 +1,46 @@
+name: Basic kb-sdk tests
+
+on:
+  [push, pull_request]
+
+jobs:
+
+  docker_build_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Add bin to $PATH
+        run: |
+          env
+          echo "::add-path::$GITHUB_WORKSPACE/bin"
+      - name: Build with Ant
+        run: |
+          make
+          env
+      - name: checking kb-sdk functions
+        env:
+          KBASE_TEST_TOKEN: ${{ secrets.KBASE_TEST_TOKEN }}
+        run: |
+          kb-sdk help
+          kb-sdk version
+          kb-sdk init -l perl -u user PerlBase
+          cd PerlBase
+          kb-sdk test
+          cd ..
+          rm -rf PerlBase
+          kb-sdk init -l perl -u user --example PerlExample
+          cd PerlExample
+          kb-sdk test
+          cd ..
+          rm -rf PerlExample
+          kb-sdk init -l python -u user --example MyContigFilter
+          cd MyContigFilter
+          make
+          kb-sdk test
+          cd ..
```

Comment:
Why no Java tests? This version of the sdk will probably be used to build Java core services for a long time, so the Java stuff is mission critical.

Reply:
I'll put them in. 👍